lean/main/09_tactics.lean

```diff
@@ -386,7 +386,7 @@ elab "custom_assump_2" : tactic =>
         then return Option.some declExpr
         else return Option.none
     match option_matching_expr with
-    | some e => Lean.Elab.Tactic.closeMainGoal e
+    | some e => Lean.Elab.Tactic.closeMainGoal `custom_assump_2 e
```

Comment:
I'm only scanning the PR so apologies for not trying to figure this out myself, but can you say something about what's happening here with the extra argument?

Reply:
The `Lean.Elab.Tactic.closeMainGoal` argument appears to have been increased. It now accepts the tactic name for use in error messages.